webapp/app/components/listProgress/listProgress.component.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { LinearProgress } from 'material-ui/Progress';
+
+
+export class ListsProgress extends PureComponent {
+  static propTypes = {
+    list: PropTypes.object,
+  };
+
+  state = {
+    value: 0,
+  };
+
+  componentDidMount() {
+    this._timeout = setTimeout(() => this.setState({ value: this.getProgressValue(this.props.list) }));
```

Comment:
Do you need this timeout?

Reply:
yup, otherwise animation doesn't work. It has to render with value 0 and change value afterwards.